airflow/models/taskinstance.py

```diff
@@ -1253,7 +1253,7 @@ class TaskInstance(Base, LoggingMixin):
     pid = Column(Integer)
     executor_config = Column(ExecutorConfigType(pickler=dill))
     updated_at = Column(UtcDateTime, default=timezone.utcnow, onupdate=timezone.utcnow)
-    rendered_map_index = Column(String(64))
+    rendered_map_index = Column(String(250))
```

Comment:
```suggestion
    rendered_map_index = Column(StringID())
```
We have that special type that equals 250. 
We should also make the changes in the migration file